### PR TITLE
Fix #517 - arity()

### LINKED ISSUE #517: fix arity method
There is currently a method called arity that lives in `/lib/arity.js`.
It is incomplete and needs to be fixed!

This function acepts a function an return the arity of the function
See: https://en.wikipedia.org/wiki/Arity

### Acceptance Criteria:
1. Running `yarn test arity` results in tests passing.
2. You have written a skeleton method for someone else to work on.
3. You have written tests surrounding your skeleton method.
4. Running `yarn lint` does not print any errors to the console!
5. Optional: write one or two more tests surrounding the method you are solving to account for potential edge cases.

### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request! A pull request will most likely be denied if it does not contain a skeleton method for someone else to work on! For more information, please read the [Contributing Guide](https://github.com/BlakeGuilloud/ganon/blob/master/CONTRIBUTING.md).

Thank you so much for your contribution!

### PR DESCRIPTION
**Pull request by SamHowie**
Fixes #517 

- [x] Running `yarn lint` does not trigger any linter errors
- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!
- [x] You have written tests to accompany your skeleton method

### CODE REVIEW

File: test/partition.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const { partition } = require("../lib");
+
+describe("partition", () => {
+  test("should partition array slices of given length", () => {
+    expect(typeof partition([1, 2, 3, 4], 1)).toEqual([[1], [2], [3], [4]]);
```
Comment (BlakeGuilloud) [inline test/partition.test.js]: Hey @SamHowie ! Thanks for your contributions. I'm a little confused on the test methods here. It looks like you are expecting a `typeof` return to equal actual values as opposed to a data type?

Comment (SamHowie) [reply test/partition.test.js]: Oh you are right this was a silly error on my part. Thanks for catching it!

Fix incoming...

Comment (BlakeGuilloud) [reply test/partition.test.js]: Thanks !

**Comment by SamHowie** [comment]
Fixed the silly errors. We should be good to go now :)

**Comment by BlakeGuilloud** [comment]
image

haven't seen this in awhile 😅

**Pull request merged**